manifests/ca.pp

```diff
@@ -106,13 +109,13 @@
           }
         }
         'ftp', 'https', 'http': {
-          remote_file { $ca_cert:
-            ensure      => present,
-            source      => $source,
-            checksum    => $checksum,
-            mode        => '0644',
-            verify_peer => $verify_https_cert,
-            notify      => Class['::ca_cert::update'],
+          archive { $ca_cert:
+            ensure         => present,
+            source         => $source,
+            checksum       => $checksum,
+            checksum_type  => $checksum_type,
+            allow_insecure => $verify_https_cert,
```

Comment:
Should this be `!$verify_https_cert` since we're re-using the `verify_peer` option?

Reply:
Yes, good catch. Fixed.